### PR TITLE
Docker update

### PR DESCRIPTION
**Pull request by adodge-bcg**
- update docker to include env vars
- update readme to include debugging tips
- moved postman files into own folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,3 +110,19 @@ Cron Expression: Cron expressions are used for more fine-tuned alert scheduling.
...
 
 Trigger Conditions: These can be used to set some additional behavior on alerts, such as how many events have to occur before an alert is triggered, if an alert should trigger once or once for each result, how much time should elapse before an alert can trigger again. 
+
+## Debugging
+
+Useful tools for testing implementation and debugging issues.
+
+### Postman
+
+There is a postman collection and environment included in the test folder. This collection can be used to test and confirm the implementation of the notification service.
```
Comment (akroon3r) [inline README.md:120]: maybe do the [test](url_to_test) for handy dandy quick linking

Comment (akroon3r) [reply README.md:120]: []()

Comment (akroon3r) [reply README.md:120]: borf

Comment (adodge-bcg) [reply README.md:120]: Wat

**Pull request merged**